vcf/vcf.go

```diff
@@ -111,6 +174,21 @@ func ReadHeader(er *fileio.EasyReader) *VcfHeader {
 	return &header
 }
 
+//FormatToString converts the []string Format struct into a string by concatenating with a colon delimiter.
+func FormatToString(format []string) string {
+	if len(format) == 0 {
+		return ""
+	}
+	if len(format) == 1 {
```

Comment:
Do we need this if statement and return?  Seems like len(format) == 1 is handled ok by the below code, which also deals with larger lengths?

Reply:
great catch!